app/examples/bitrise/github-release.yml

```diff
@@ -0,0 +1,55 @@
+format_version: 11
+default_step_lib_source: https://github.com/bitrise-steplib/steps-github-release.git
+project_type: ios
+
+workflows:
+  build:
+    steps:
+    - git-clone@8.2.2:
+        inputs:
+        - clone_into_dir: /tmp/bitrise-ex/
+        - repository_url: git@github.com:example/example-repository.git
+    - script@1.1.5:
+        inputs:
+        - content: ./build-app.sh
+    - github-release@3.0.1:
+        inputs:
+        - api_token: $GITHUB_TOKEN
+        - username: example
+        - repository_url: git@github.com:example/example-repository.git
+        - tag: v1.0.0
+        - commit: $BITRISE_GIT_COMMIT
+        - name: Release v1.0.0
+        - body: This is the release body.
+        - draft: no
+        - pre_release: no
+        - files_to_upload: |
+            /tmp/bitrise-ex/artifacts/app-debug.apk|app-debug.apk
+            /tmp/bitrise-ex/artifacts/app-debug-androidTest.apk|app-debug-androidTest.apk
```

Comment:
as per the original documentation I would be interested in seeing a couple more test cases:

* when no alternative filename is provided
* when the original filename contains a variable
* when the proposed filename contains a variable

Reply:
this still applies